src/rich-log/MessageBuilder.hh

```diff
@@ -14,20 +14,29 @@ namespace rlog
 class MessageBuilder
 {
     // options
-private:
+public:
     void configure(prefix const& p) { _prefix = p.value; }
-    void configure(channel c) { _channel = c; }
     void configure(sep const& s) { _sep = s.value; }
     void configure(no_sep_t) { _sep = ""; }
+    void configure(err_out_t) { _use_err_stream = true; }
+    void configure(location const& loc) { _location = &loc; }
```

Comment:
do we want to make this one private?

Reply:
actually see comment regarding MessageBuilder ctor, I think we don't want this configure